### ISSUE TITLE
iOS dynamic text exceeds size of unresponsive buttons

### ISSUE THREAD

**Issue by juliaxbow** [berty/berty#4276]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Berty product

Mobile app

### Berty product version

2.391.3

### OS

iOS

### OS version

iOS 15.5

### Device

iPhone 12 Pro

### Steps to reproduce

1. Access phone's accessibility settings and increase text size
2. Open Berty App
3. Create an account
4. Navigate to settings

### Current behavior

Text in the Berty app is dynamic however the buttons and cards do not appear to be responsive. E.g.: if the user's phone is set to a large text size, the text in the app exceeds the size of buttons and cards.

**Create account page:** 
![IMG_2960](https://user-images.githubusercontent.com/58958327/178565277-bc4ac9bb-5f56-47dd-8ca5-c3ac941bc8b8.PNG)
![IMG_2962](https://user-images.githubusercontent.com/58958327/178565511-45128ab1-50fc-4a9e-b098-2496faeffaa2.PNG)

**Settings page:**
![IMG_2965](https://user-images.githubusercontent.com/58958327/178565387-f66ed0db-09ae-4f15-885c-932db359bb05.PNG)

### Expected behavior

Buttons and text blocks should be responsive and fit the full text; if the full text does not fit, text should be scrollable.

**Comment by peletron** [comment]
@juliaxbow Thanks for the reporting. We are going to take a look at the issue. Can you please tell me one more piece of information? Under your **Settings -> Display & Brightness -> Text size** , is text font size increased or it is default one?

**Comment by juliaxbow** [comment]
@peletron yes, the text size is increased. Submitting the issue only for larger text sizes as there was no bug with default text settings (where buttons and text appear normal)

**Comment by jefft0** [comment]
@iuricmp confirms this behavior if you increase the font size to the maximum. It would take some time to fix. Put in the backlog for now.

**Comment by cktsmith** [comment]
I also see this issue, with the font size set to HALFWAY (not the maximum, which jefft0 mentioned testing). None of my other apps have button and interface text as large as what I see in Berty. My screens appear like those above in screenshots from Juliaxbow, but my system-wide Dynamic Type textsize slider is in the center (halfway between the extremes).

I'm using Berty v2.470.0 with iOS 17.4.1
